crypto/ctype.c

```diff
@@ -272,3 +272,10 @@ int ossl_toupper(int c)
 {
     return ossl_islower(c) ? c ^ case_change : c;
 }
+
+int ascii_isdigit(const char inchar) {
+    if (inchar > 0x2F && inchar < 0x3A)
```

Comment:
What about `return inchar >= 060 && inchar <= 072;` or equivalent here?

Reply:
I would really like to keep these in hex as all the other ASCII/EBCDIC charset issues are expressed that way.